Patches/Revia Race/ThingDefs_Misc/Weapons_Guns.xml

```diff
@@ -0,0 +1,634 @@
+                    <armorPenetrationSharp>0.74</armorPenetrationSharp>
+                  </li>
+                </tools>
+              </value>
+            </li>
+            <li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+              <defName>ReviaFlechetteCannon</defName>
+              <statBases>
+                <SightsEfficiency>1.0</SightsEfficiency>
+                <Bulk>10.0</Bulk>
+                <Mass>8.0</Mass>
+                <ShotSpread>0.18</ShotSpread>
+                <SwayFactor>1.45</SwayFactor>
+                <RangedWeapon_Cooldown>1.15</RangedWeapon_Cooldown>
+              </statBases>
+              <costList>
+                <Steel>10</Steel>
+              </costList>
+              <Properties>
+                <verbClass>CombatExtended.Verb_ShootCE</verbClass>
+                <hasStandardCommand>True</hasStandardCommand>
+                <defaultProjectile>Bullet_12Gauge_Buck</defaultProjectile>
+                <warmupTime>0.6</warmupTime>
+                <range>21</range>
+                <recoilAmount>2.35</recoilAmount>
+                <burstShotCount>3</burstShotCount>
+                <soundCast>Shot_Shotgun</soundCast>
+                <soundCastTail>GunTail_Heavy</soundCastTail>
+                <muzzleFlashScale>9</muzzleFlashScale>
+                <ai_IsBuildingDestroyer>false</ai_IsBuildingDestroyer>
+              </Properties>
+              <weaponTags>
+                <li>ReviaRangedAdvanced</li>
+              </weaponTags>
+              <AmmoUser>
+                <magazineSize>8</magazineSize>
+                <reloadTime>4.8</reloadTime>
+                <ammoSet>AmmoSet_12Gauge</ammoSet>
+              </AmmoUser>
+              <FireModes>
+                <aiUseBurstMode>FALSE</aiUseBurstMode>
+                <aiAimMode>Snapshot</aiAimMode>
+                <aimedBurstShotCount>2</aimedBurstShotCount>
+              </FireModes>
+            </li>
+            <!-- ==========  Melee  =========== -->
+            <!-- ==========  Eviscerator  =========== -->
+            <li Class="PatchOperationReplace">
+              <xpath>Defs/ThingDef[defName="ReviaSerratedDagger"]/tools</xpath>
+              <value>
+                <tools>
+                  <li Class="CombatExtended.ToolCE">
+                    <label>handle</label>
+                    <capacities>
+                      <li>Blunt</li>
+                    </capacities>
+                    <power>7</power>
+                    <cooldownTime>1.55</cooldownTime>
+                    <armorPenetrationBlunt>0.15</armorPenetrationBlunt>
+                    <linkedBodyPartsGroup>Handle</linkedBodyPartsGroup>
+                  </li>
+                  <li Class="CombatExtended.ToolCE">
+                    <label>point</label>
+                    <capacities>
+                      <li>Stab</li>
+                    </capacities>
+                    <power>12</power>
```

Comment:
As something that's odd with a lot of your edged weapons, Stab attacks should generally do less damage but have more armor penetration. Check out the Melee stat sheet linked on the Github wiki if you haven't already--the stats for the vanilla weapons there may be insightful.